ci-operator/config/openshift/dpu-operator/openshift-dpu-operator-main.yaml

```diff
@@ -41,10 +41,11 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: unit
```

Comment:
Please don't remove this unit test. Make a new one

Reply:
ack